cegs_portal/search/views/v1/search.py

```diff
@@ -469,7 +487,7 @@ def get_tsv(self, request, options, data):
             filename = f"significant_reos_{region.chromo}_{region.range.lower}_{region.range.upper}.tsv"
         return super().get_tsv(request, options, data, filename=filename)
 
-    def get_data(self, options):
+    def get_data(self, options) -> Pageable[RegulatoryEffectObservation]:
```

Comment:
This type signature is no longer correct. Deleting it will also let you revert some of the changes to the imports.